frontend/src/global/constants.ts

```diff
@@ -9,13 +9,15 @@ export const ENTERPRISE_COMPARISON_LINK =
 export const TOS_LINK = process.env.REACT_APP_TOS_LINK || '/#';
 export const PRIVACY_POLICY_LINK =
   process.env.REACT_APP_PRIVACY_POLICY_LINK || '/#';
+export const FAQ_PAGE_LINK = process.env.REACT_APP_FAQ_PAGE_LINK || '';
```

Comment:
@mavidser  How do I add something to the `process.env`? I was able to test this change by modifying this file to make the default be non-empty, but am not sure how this change will actually be deployed once we have a URL.

Reply:
`process.env` picks up environment variables - and from what I gather - the UI is built and deployed by circleCI, so we would have to add it in their dashboard most likely. @giovannicimolin Can you confirm this?

Reply:
Ah cool yep, it was enough to run `REACT_APP_FAQ_PAGE_LINK="https://opencraft.com" npm start` to set that env variables here.

And yep, the CircleCI project has a full list of env variables, so we'd just need to add that here.

Reply:
@pomegranited @mavidser The frontend variables are set using CircleCI contexts, here: https://circleci.com/gh/organizations/open-craft/settings#contexts
This allow us to use the same build roles to build stage and prod (just changing the context).